hack/cluster-deploy.sh

```diff
@@ -46,6 +46,28 @@ function _ensure_cdi_deployment() {
     # Do not deploy any cdi-operator related objects on
     # sriov-lane until kubevirt/kubevirt#4120 is fixed
     if [[ ! "$KUBEVIRT_PROVIDER" =~ sriov.* ]]; then
+        # Prefetch cdi images
+        container=""
+        cdi_images=$(sed -ne "s/.*\(quay.io[^\s]*\)/\1/p" manifests/testing/cdi*)
+        for arg in ${cdi_images}; do
+            container="${container} ${arg}"
+        done
+
+        pull_command=$(find_pull_command)
```

Comment:
You may want  to make a function out of this, since this is now also repeated. Something like


```
function pull_on_nodes() {
containers_to_pull=$@
 ...
}
```

and call it then like this:

```
cdi_images=$(sed -ne "s/.*\(quay.io[^\s]*\)/\1/p" manifests/testing/cdi*)
pull_on_nodes $cdi_images
```

and place it in the new common script which I mention in another comment.

Reply:
done